src/main/steps/sequence.ts

```diff
@@ -0,0 +1,172 @@
+import { Request } from 'express';
+
+import {
+  HAS_RELATIONSHIP_BROKEN_URL,
```

Comment:
I wonder if these will be needed in this new system?

Reply:
I did think about moving all the URL here, but some of the tests still need them, so I've left them where there were.